packages/app-check/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/appcheck/FirebaseAppCheckPlugin.java

```diff
@@ -45,9 +45,18 @@ public void error(String message) {
     @PluginMethod
     public void initialize(PluginCall call) {
         try {
-            boolean debug = call.getBoolean("debug", false);
+            Boolean debugTokenBoolean = call.getBoolean("debugToken", false);
+
+            boolean hasDebugToken;
+            if (Boolean.TRUE.equals(debugTokenBoolean)) {
+                hasDebugToken = true;
+            } else {
+                String debugTokenString = call.getString("debugToken", "");
+                hasDebugToken = debugTokenString != null && !debugTokenString.isEmpty();
```

Comment:
```suggestion
                String debugTokenString = call.getString("debugToken");
                hasDebugToken = debugTokenString != null;
```

Reply:
How would you match the empty string then? In web it is seen as falsy, but in this case in Android code `debugTokenString != null` will return as `true` if `debugToken` option is set to `""`.